neqo-transport/src/cc/classic_cc.rs

```diff
@@ -315,7 +316,7 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
         );
     }
 
-    fn on_packet_sent(&mut self, pkt: &SentPacket) {
+    fn on_packet_sent(&mut self, pkt: &SentPacket, rtt: Duration) {
```

Comment:
Is there a better way to get at the RTT information other than passing it around the stack?

Reply:
Not without doing something gross with `Rc`.  That's a consequence of rust being rust.